examples/flogo/creditcard/creditcard_test.go

```diff
@@ -0,0 +1,334 @@
+	payload = `{
+		"Name": "Sam",
+		"Age": "26",
+		"Income": "50100",
+		"Address": "SFO",
+		"Id": "4"
+	  }`
+	payload1 = `{
+		"Id": "4",
+		"creditScore": "850"
+	  }`
+	payload2 = `{
+		"Name": "Sam1",
+		"Age": "17",
+		"Income": "50100",
+		"Address": "SFO",
+		"Id": "5"
+	  }`
+	payload3 = `{
+		"Name": "Sam2",
+		"Age": "26",
+		"Income": "5100",
+		"Address": "SFO",
+		"Id": "6"
+	  }`
+	payload4 = `{
+		"Name": "Sam3",
+		"Age": "26",
+		"Income": "5100",
+		"Address": "",
+		"Id": "7"
+	  }`
+	payload5 = `{
+		"Name": "Sam5",
+		"Age": "32",
+		"Income": "75000",
+		"Address": "SFO",
+		"Id": "8"
+	  }`
+	payload6 = `{
+		"Id": "8",
+		"creditScore": "760"
+	  }`
+	payload7 = `{
+		"Name": "Sam4",
+		"Age": "41",
+		"Income": "30000",
+		"Address": "SFO",
+		"Id": "9"
+	  }`
+	payload8 = `{
+		"Id": "9",
+		"creditScore": "720"
+	  }`
+)
+
+func testapplication(t *testing.T, e engine.Engine) {
+	tests.Drain("7777")
+	err := e.Start()
+	assert.Nil(t, err)
+	defer func() {
+		err := e.Stop()
+		assert.Nil(t, err)
+	}()
+	tests.Pour("7777")
+	transport := &http.Transport{
+		MaxIdleConns: 1,
+	}
+	defer transport.CloseIdleConnections()
+	client := &http.Client{
+		Transport: transport,
+	}
+
+	payload, err := json.Marshal(payload)
+	if err != nil {
+		panic(err)
+	}
+
+	payload1, err := json.Marshal(payload1)
+	if err != nil {
+		panic(err)
+	}
+
+	payload2, err := json.Marshal(payload2)
+	if err != nil {
+		panic(err)
+	}
+
+	payload3, err := json.Marshal(payload3)
+	if err != nil {
+		panic(err)
+	}
+
+	payload4, err := json.Marshal(payload4)
+	if err != nil {
+		panic(err)
+	}
+
+	payload5, err := json.Marshal(payload5)
+	if err != nil {
+		panic(err)
+	}
+
+	payload6, err := json.Marshal(payload6)
+	if err != nil {
+		panic(err)
+	}
+
+	payload7, err := json.Marshal(payload7)
+	if err != nil {
+		panic(err)
+	}
+
+	payload8, err := json.Marshal(payload8)
+	if err != nil {
+		panic(err)
+	}
+
+	//  valid new user details
+	request := func() {
+		req, err := http.NewRequest(http.MethodPut, "http://localhost:7777/newaccount", bytes.NewBuffer(payload))
+		assert.Nil(t, err)
+		req.Header.Set("Content-Type", "application/json")
+		response, err := client.Do(req)
+		assert.Nil(t, err)
+		_, err = ioutil.ReadAll(response.Body)
+		assert.Nil(t, err)
+		response.Body.Close()
+		assert.Nil(t, err)
+	}
+	outpt := tests.CaptureOutput(request)
+	var result string
+	if strings.Contains(outpt, "Rule fired") {
+		result = "success"
+	}
+	assert.Equal(t, "success", result)
+	outpt = ""
+	result = ""
+
+	//  user with credit score > 850
+	request1 := func() {
+		req, err := http.NewRequest(http.MethodPut, "http://localhost:7777/credit", bytes.NewBuffer(payload1))
+		assert.Nil(t, err)
+		req.Header.Set("Content-Type", "application/json")
+		response, err := client.Do(req)
+		assert.Nil(t, err)
+		_, err = ioutil.ReadAll(response.Body)
+		assert.Nil(t, err)
+		response.Body.Close()
+		assert.Nil(t, err)
+	}
+	outpt = tests.CaptureOutput(request1)
```

Comment:
@LakshmiMekala I feel like Capture**Std**Output() would be more appropriate name than CaptureOutput() as it is capturing standard output.

Reply:
required changes are done